README.md

```diff
@@ -79,32 +77,36 @@ You may download the `sp.js` file from the Releases section Github, self-host it
 To track events, simply call their corresponding functions given the event data:
 
 ```javascript
-trackSelfDescribingEvent({
```

Comment:
Nit: I wouldn't remove the `trackSelfDescribingEvent` function from the docs here – that is still the easiest to use (having to set the `eventName` to `ue` is not very intuitive). The `trackWebViewEvent` function is mostly useful for us in the JS tracker so that we can pass the whole event to the WebView tracker, but I expect users of the WebView tracker could still find the other ones easier to use.